### PR TITLE
Adding my Kyria/rev1 keymap

### PR DESCRIPTION
**Pull request by ghidalgo93**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/kyria/keymaps/ghidalgo93/keymap.c [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+            break;
+        default:
+            oled_write_P(PSTR("Undefined\n"), false);
+    }
+
+    // Host Keyboard LED Status
+    uint8_t led_usb_state = host_keyboard_leds();
+    oled_write_P(IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK) ? PSTR("NUMLCK ") : PSTR("       "), false);
+    oled_write_P(IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK) ? PSTR("CAPLCK ") : PSTR("       "), false);
+    oled_write_P(IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/ghidalgo93/keymap.c:214]:
```suggestion
    led_t led_state = host_keyboard_led_state();
    oled_write_P(led_state.num_lock ? PSTR("NUMLCK ") : PSTR("       "), false);
    oled_write_P(led_state.caps_lock ? PSTR("CAPLCK ") : PSTR("       "), false);
    oled_write_P(led_state.scroll_lock ? PSTR("SCRLCK ") : PSTR("       "), false);
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kyria/keymaps/ghidalgo93/keymap.c [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+ * ,-------------------------------------------.                              ,-------------------------------------------.
+ * |   ESC  |   Q  |   W  |   E  |   R  |   T  |                              |   Y  |   U  |   I  |   O  |   P  |  | \   |
+ * |--------+------+------+------+------+------|                              |------+------+------+------+------+--------|
+ * |  Ctrl  |   A  |   S  |  D   |   F  |   G  |                              |   H  |   J  |   K  |   L  | ;  : |  ' "   |
+ * |--------+------+------+------+------+------+-------------.  ,-------------+------+------+------+------+------+--------|
+ * | LShift |   Z  |   X  |   C  |   V  |   B  |      |      |  |      |      |   N  |   M  | ,  < | . >  | /  ? |  - _   |
+ * `----------------------+------+------+------+------+------|  |------+------+------+------+------+----------------------'
+ *                        |      |  GUI	|      | Tab  | Enter|  | Bksp | Space| Alt  | Home | End  |
+ *                        |      |      | LCTL | Lower|      |  | 	   | Raise|      |      |      |
+ *                        `----------------------------------'  `----------------------------------'
```
Comment (precondition) [inline keyboards/kyria/keymaps/ghidalgo93/keymap.c:40]:
```suggestion
 *                        |      |  GUI |      | Tab  | Enter|  | Bksp   | Space| Alt  | Home | End  |
 *                        |      |      | LCTL | Lower|      |  | 	   | Raise|      |      |      |
 *                        `----------------------------------'  `----------------------------------'
```

**Review by ghidalgo93 — COMMENTED** [review]
adding suggested changes

File: keyboards/kyria/keymaps/ghidalgo93/keymap.c [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+ *
+ * ,-------------------------------------------.                              ,-------------------------------------------.
+ * |   ESC  |   Q  |   W  |   E  |   R  |   T  |                              |   Y  |   U  |   I  |   O  |   P  |  | \   |
+ * |--------+------+------+------+------+------|                              |------+------+------+------+------+--------|
+ * |  Ctrl  |   A  |   S  |  D   |   F  |   G  |                              |   H  |   J  |   K  |   L  | ;  : |  ' "   |
+ * |--------+------+------+------+------+------+-------------.  ,-------------+------+------+------+------+------+--------|
+ * | LShift |   Z  |   X  |   C  |   V  |   B  |      |      |  |      |      |   N  |   M  | ,  < | . >  | /  ? |  - _   |
+ * `----------------------+------+------+------+------+------|  |------+------+------+------+------+----------------------'
+ *                        |      |  GUI |      | Tab  | Enter|  | Bksp   | Space| Alt  | Home | End  |
+ *                        |      |      | LCTL | Lower|      |  | 	   | Raise|      |      |      |
```
Comment (noroadsleft) [inline keyboards/kyria/keymaps/ghidalgo93/keymap.c:39]: Alignment fix

```suggestion
 *                        |      |  GUI |      | Tab  | Enter|  | Bksp | Space| Alt  | Home | End  |
 *                        |      |      | LCTL | Lower|      |  |      | Raise|      |      |      |
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Pull request merged**